front/app/modules/commercial/content_builder/admin/components/ContentBuilderTopBar/index.tsx

```diff
@@ -99,6 +99,7 @@ const ContentBuilderPage = ({ params: { projectId } }) => {
           )}
         </Box>
         <Button
+          id="contentBuilderTopBarSaveButton"
```

Comment:
Would be nice to also keep them consistently formatted - either with dashes or camelCase. Not sure if we have a convention now that we prefer by maybe looking at other examples might help decide.